.dev-deploy/web-connect.Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Stage 1: Build Stage
+FROM node:20.11.0 AS build
```

Comment:
any reason to not also use the -alpine image here?

Reply:
Just to make sure if you are actually looking at that code :P

I have updated to use alpine, I can't see any reason why it should not be node-alpine here.